athenet/utils/count_n_elts.py

```diff
@@ -0,0 +1,88 @@
+"""Function that for given neural network, returns lists of numbers of edges,
+weights and neurons per layer."""
+
+from  athenet.models import *
```

Comment:
not really nice way of imports, even pep8 says it should be avoided (https://www.python.org/dev/peps/pep-0008/#imports)
few times I get some errors from too much imports like this